examples/Interpolating/validation_spline_interp_with_MC.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
As mentioned before, I would love to take that as an example into the PyDynamic_tutorials. In that case by the way, we would already have a test against expected output. We could discuss that further in the Code Review.

Reply:
I have prepared something regarding this 😃

Reply:
Nice!